### DISCUSSION TITLE
Docker container rewriting url to include wrong port.

### DISCUSSION THREAD

**Discussion by ghost** [firefly-iii/firefly-iii#5560] [Firefly III] [answered]
I went through my usual weekly docker container updates and now for firefly-iii when connecting it rewrites the URL to include the docker port of 8080, which is the same as the host port of 8080. I have been proxying it behind nginx and nothing else changed. Its very possible something I did messed it up but just wanted to make sure I am not missing any changes to the docker image in 5.6.10.

I am using the docker-compose method which since I am using Portainer is version 3.3 I believe.

**Comment by JC5** [comment, marked as answer]
Nothing really changed in the Docker container; did you come from 5.6.9 or an older version?

**Reply by ghost** [reply]
Yeah, moved from 5.6.9. To do so I deleted both containers core and mariadb. Recreated from docker-compose which is how I have done it many times. Again it must be something else but nothing that I can think of changed.

**Reply by ghost** [reply]
As an update, the issue turned out to be an error introduced in Nginx-Proxy-Manager that was fixed on a new update. Marking as answered but the true issue never was related to Firefly-iii. I do not know why that was the only impacted service but moot point at this juncture. Thank you for the help!

**Reply by JC5** [reply]
Thanks for letting me know!
